requirements.txt

```diff
@@ -23,3 +23,4 @@ nbmake
 pennylane-lightning[kokkos]
 amazon-braket-pennylane-plugin>=1.23.0
 lark
+optax
```

Comment:
Hey @maliasadi @dime10, any idea why this is not fixing the CI issue with the new demo? 🤔 

https://github.com/PennyLaneAI/catalyst/actions/runs/8362374850/job/22893066971?pr=612#step:11:36

As far as I can tell, `requirements.txt` is used for this workflow.

Reply:
solved!